yt_dlp/extractor/bilibili.py

```diff
@@ -1489,6 +1489,88 @@ def _real_extract(self, url):
         return self.playlist_result(self._entries(category, subcategory, query), query, query)
 
 
+class BiliBiliSearchAllIE(SearchInfoExtractor):
+    IE_DESC = 'Bilibili all search'
+    _MAX_RESULTS = 100000
+    _SEARCH_KEY = 'bilisearchall'
```

Comment:
>```py
>     @classproperty
>    def _VALID_URL(cls):
>        return rf'{cls._SEARCH_KEY}(?P<prefix>|[1-9][0-9]*|all):(?P<query>[\s\S]+)'
>```

([common.py:3947]())
I just need to change the `_SEARCH_KEY`. `all` is matched as a prefix in `_VALID_URL`.